native-rendering/documents-designer/data-source.md

```diff
@@ -14,6 +14,13 @@ or the bracket notation:
 ```code
 $['shipment']['tracking_number']
 ```
+
+For each document you need to select DataSource property. Select a 'package' if you want to print label for each package, and 'shipment' if you want only one label for whole shipment.
```

Comment:
```suggestion
For each document you need to select DataSource property value:
* **package** -  prints a label for each package in the shipment;
* **shipment** - prints only one label for the whole shipment.
```
To consider, and check whether formats and looks nice. Bold values are consistent with listings on  DesignerWindow page